screens/Contact/Contact.styles.js

```diff
@@ -0,0 +1,26 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+    layout: {
+        flex: 1,
+        justifyContent: 'center',
+        alignItems: 'center',
+        backgroundColor: '#00FA9A',
+    },
+    input: {
+        marginTop: 0,
+        marginLeft: 10,
+        marginRight: 10,
+        backgroundColor: '#FFFFFF',
```

Comment:
Max white color (and black) is discouraged since it could strain the eyes. Maybe try "#FAFAFA" or something.

Reply:
Might have to change all pure white (and black) colors too in other files too.